docs/java-development.rst

```diff
@@ -110,19 +110,33 @@ Release process
 Maintainer prerequisites
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-It is important to get familiar with the
-`OSSRH guide <https://central.sonatype.org/pages/ossrh-guide.html>`__ and
-especially the
-`Maven section for performing a release deployment <https://central.sonatype.org/pages/apache-maven.html>`__.
+To be able to maintain a Java component, a developer must have:
 
-To be able to maintain a Java component, a developer must:
+- a GitHub_ account with push rights to the GitHub source code repository
+- a Sonatype_ account and be registered as a maintainer of the
```

Comment:
This does not apply to the omero Java component e.g. omero-model

Reply:
Yes, I believe the name of the page is slightly misleading. Its content applies only to the OME Java libraries, built using Maven and deployed to Central. The first section gives a table of the relevant components but I would be opened to updating the name of the page if that helps clarifying the scope.